src/utils/yarn.js

```diff
@@ -3,11 +3,17 @@ import includes from 'array-includes';
 
 import type { Dependency, configDependencyType } from '../types';
 import type Package from '../Package';
+import Project from '../Project';
 import * as processes from './processes';
 import * as fs from '../utils/fs';
 import * as logger from '../utils/fs';
 import { DEPENDENCY_TYPE_FLAGS_MAP } from '../constants';
 
+async function getProjectBinPath(cwd: string): Promise<string> {
+  const project = await Project.init(cwd);
+  return project.pkg.nodeModulesBin;
```

Comment:
Had to revert this in #70 because this should be pulling from the bin path of `bolt` not the project.